### PR TITLE
Slack install button and auth flow

### PR DESCRIPTION
**Pull request by shepwalker**

### CODE REVIEW

File: corji/static/css/style.css [hunk 1]
```diff
@@ -12,6 +12,23 @@ h1 {
...
 }
 
+h1.corgibot {
+    font: 64px Avenir;
+    margin-bottom: 0;
+    text-align: center;
+}
+
+h2.corgibot {
+    font-family: 'Sacramento';
```
Comment (jmduke) [inline corji/static/css/style.css]: isn't this a google font

File: corji/views/slack.py [hunk 2]
```diff
@@ -82,4 +85,21 @@ def slack_corgi():
...
-    return Response(response_content, mimetype='application/json')
\ No newline at end of file
+    return Response(response_content, mimetype='application/json')
+
+
+@slack_blueprint.route("/slack/about", methods=['GET'])
+def about_slack():
+    code = request.args.get('code','')
+    redirect_uri = url_for("slack.about_slack", _external=True)
+    if(code):
```
Comment (jmduke) [inline corji/views/slack.py]: `if code:`

File: corji/views/slack.py [hunk 3]
```diff
@@ -82,4 +85,21 @@ def slack_corgi():
...
+        auth_response = requests.post('https://slack.com/api/oauth.access',
+                              data ={
```
Comment (jmduke) [inline corji/views/slack.py]: `data={`

File: corji/views/slack.py [hunk 4]
```diff
@@ -82,4 +85,21 @@ def slack_corgi():
...
+                                  'client_id': Config.SLACK_ID,
+                                  'client_secret': Config.SLACK_SECRET,
+                                  'code': code,
+                                  'redirect_uri':redirect_uri
```
Comment (jmduke) [inline corji/views/slack.py]: consistent spacing

File: corji/models/slack_customer.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+# Normally this should be something more specific -- but legacy reasons.
+TABLE_NAME = "corji_slack"
+
+
+def get(team_id):
+    response = dynamo_client.get_item(
+        TableName=TABLE_NAME,
+        Key={'team_id': {'S': team_id}}
+    )
+    if 'Item' in response:
```
Comment (jmduke) [inline corji/models/slack_customer.py]: this can be rewritten as `response.get('Item', None)`

File: corji/views/slack.py [hunk 6]
```diff
@@ -82,4 +86,31 @@ def slack_corgi():
...
+    if code:
+        auth_response = requests.post('https://slack.com/api/oauth.access',
+                              data ={
+                                  'client_id': Config.SLACK_ID,
+                                  'client_secret': Config.SLACK_SECRET,
+                                  'code': code,
+                                  'redirect_uri':redirect_uri
+                              })
+        response_content = json.loads(auth_response.text)
+        print(response_content)
```
Comment (jmduke) [inline corji/views/slack.py]: remove dis

File: corji/static/css/style.css (hunk 1 above)
Comment (shepwalker) [inline corji/static/css/style.css]: ? I think I got all the font style stuff from existing css.

Comment (jmduke) [inline corji/static/css/style.css]: ah kk

**Pull request merged**